### PR TITLE
update(v6-migration): ensemble matching

### PR DESCRIPTION
**Pull request by zakkarry**
placeholder

### CODE REVIEW

File: docs/basics/options.md [hunk 1]
```diff
@@ -736,6 +738,29 @@ includeSingleEpisodes: true,
 includeSingleEpisodes: false,
 ```
 
+### `seasonFromEpisodes`
+
+| Config file name     | CLI short form | CLI long form            | Format    | Default |
+| -------------------- | -------------- | ------------------------ | --------- | ------- |
+| `seasonFromEpisodes` | `N/A`          | `--season-from-episodes` | `boolean` | `false` |
+
+Set this to `true` to aggregate/join single episodes to season packs when they are sent to `cross-seed` for search or RSS.
```
Comment (ShanaryS) [inline docs/basics/options.md:747]:
```suggestion
When `true`, `cross-seed` will also aggregate individual episodes into season packs for searching (when applicable) or to match with season packs from rss/announce.
```

File: docs/basics/options.md [hunk 2]
```diff
@@ -736,6 +738,29 @@ includeSingleEpisodes: true,
...
+
+#### `seasonFromEpisodes` Examples (CLI)
+
+```shell
+cross-seed search --season-from-episodes # will combine episodes into season packs if possible
```
Comment (ShanaryS) [inline docs/basics/options.md:752]:
```suggestion
cross-seed search --season-from-episodes # will also combine episodes into season packs if possible
```

File: docs/v6-migration.md [hunk 3]
```diff
@@ -200,6 +202,12 @@ The series or movie _must be added in your instance of Sonarr or Radarr._ You **
 **We do not query any external metadata servers.**
 :::
 
+#### Ensemble or "Torrent Aggregation"
+
+Many of you may be familiar with things like [seasonpackarr](https://github.com/nuxencs/seasonpackarr) which - for qBittorrent - will aggregate/join your already downloaded episodes into season packs when they are combined and uploaded on your trackers, and inject them. `cross-seed` can now do the same, and potentially more, linking and then creating a "combined" set of matching torrents or data to get you seeding data you already have faster. This applies to things like season packs, but also could be used in other cases like anime "seasons" or multi-season packs.
+
+This functionality for season packs is linked to the new option, `seasonFromEpisodes` - which can be set to either `true` or `false`.
```
Comment (ShanaryS) [inline docs/v6-migration.md:209]:
```suggestion
Many of you may be familiar with things like [seasonpackarr](https://github.com/nuxencs/seasonpackarr) which - for qBittorrent - will aggregate/join your already downloaded episodes into season packs when they are combined and uploaded on your trackers, and inject them. `cross-seed` can now do the same, linking and then creating a "combined" set of matching torrents or data to get you seeding data you already have faster. Currently, only season packs from individual episodes is supported but we may support additional aggregation in the future.

This functionality for season packs is linked to the new option, `seasonFromEpisodes` - which can be set to either `true` or `false`.

:::tip
This feature works best with `matchMode: "partial"` and [Sonarr](./basics/options.md#sonarr).
:::
```

**Review by ShanaryS — CHANGES_REQUESTED** [review]

File: docs/v6-migration.md (hunk 3 above)
Comment (zakkarry) [reply docs/v6-migration.md:209]: Maybe the 'for qbittorrent' line is a bit misleading that this also only works for qbittorrent?

Comment (ShanaryS) [reply docs/v6-migration.md:209]: Yeah it probably isn't necessary. Though I think it should be clear it doesn't apply to cross-seed.

**Review by ShanaryS — APPROVED** [review]

**Pull request merged**